tensorflow/python/ops/nn_ops.py

```diff
@@ -297,6 +297,68 @@ def conv2d_transpose(value,
                                             name=name)
 
 
+def conv3d_transpose(value,
+                     filter,
+                     output_shape,
+                     strides,
+                     padding="SAME",
+                     name=None):
+  """The transpose of `conv3d`.
```

Comment:
Add a comment like with conv2d_transpose that this is sometimes referred to in neural network literature as 'deconvolution'.